### PR TITLE
Add Cumulus archive and Cumulus ingest to Cumulus output module

### PR DESCRIPTION
**Pull request by amarouane-ABDELHAK**
**Summary:** Summary of changes

Allows Cumulus users to request the variables or resources ARNs used bu Cumulus module via `module.cumulus.cumulus_archive.<resource>` or  `module.cumulus.cumulus_ingest.<resource>`
## Changes

* Adding the output section for ingest and archive
## PR Checklist

- [ ] Update CHANGELOG
- [ ] Unit tests
- [ ] Ad-hoc testing - Deploy changes and test manually
- [ ] Integration tests

### CODE REVIEW

**Comment by Jkovarik** [comment]
Thanks for the contribution, we'll take a look!

File: tf-modules/cumulus/outputs.tf [hunk 1]
```diff
@@ -169,3 +169,16 @@ output "ecs_cluster_name" {
 output "start_sf_queue_url" {
   value = module.ingest.start_sf_queue_url
 }
+
+
+# Ingest
+
+output "cumulus_ingest" {
```
Comment (Jkovarik) [inline tf-modules/cumulus/outputs.tf:176]: I'm not certain we're going to want to implicitly expose everything the ingest module is providing over being explicit about the specific outputs, much for the same reasons APIs make deliberate choices to expose outputs.    We don't want to create a situation where refactors (such as making the modules more....modular) result in significant user breaking changes.     I think that's worth a discussion before this is approved.

**Review by Jkovarik — CHANGES_REQUESTED** [review]
Requesting a discussion before approval.

**Comment by amarouane-ABDELHAK** [comment]
Breaking variables are expected anyway whenever there is a new upgrade, if Cumulus stopped mirroring something in ingest for example they can put it in a breaking change, an integrator would know since terraform will fail during `tf plan` and show you an output is not mirrored. 
I am not saying delete the straightforward mirrored variables, but sometimes we really need some variables and it won't make sense to add PRs just for each variable needed in the future and wait x months to get it in a release.

**Comment by Jkovarik** [comment]
closing to remove from active PR list.  Feel free to reopen for discussion.

**Pull request closed without merging**